addons/website_sale/tests/test_website_sale_pricelist.py

```diff
@@ -558,6 +560,15 @@ def test_get_pricelist_available_geoip4(self):
             pls = self.website.get_pricelist_available(show_visible=True)
         self.assertEqual(pls, pls_to_return + current_pl, "Only pricelists for BE, accessible en website and selectable should be returned. It should also return the applied promo pl")
 
+    def test_get_pricelist_available_geoip5(self):
+        # Test get all available pricelists with geoip for a country not existing in any pricelists
+
+        self.website1_be_pl += self.env.user.partner_id.property_product_pricelist
```

Comment:
does this really change something ?
Seems like `self.website1_be_pl` is not used here :thinking: